frontend/src/components/auth/Signup.tsx

```diff
@@ -29,7 +42,7 @@ const Signup = (): React.ReactElement => {
 
   return (
     <div style={{ textAlign: "center" }}>
-      <h1>Signup</h1>
+      <h1>{role.charAt(0).toUpperCase() + role.slice(1)} Signup</h1>
```

Comment:
same as above - after merging in the latest `main` branch you can use the util function in `frontend/src/utils/StringUtils.ts` :D